pyproject.toml

```diff
@@ -55,6 +55,18 @@ readme = "README.md"
 requires-python = ">=3.9"
 
 [project.optional-dependencies]
+GDS_cu11 = [
+  "kvikio-cu11>=25.02.01",
+  "dataclasses",
+  "functools",
+  "operator"
```

Comment:
These 3 are modules in the python standard library. You don't have to list them as dependencies